apis/v1alpha1/opentelemetrycollector_types.go

```diff
@@ -164,7 +164,7 @@ type OpenTelemetryTargetAllocatorSpec struct {
 // +kubebuilder:printcolumn:name="Version",type="string",JSONPath=".status.version",description="OpenTelemetry Version"
 // +kubebuilder:printcolumn:name="Age",type="date",JSONPath=".metadata.creationTimestamp"
 // +operator-sdk:csv:customresourcedefinitions:displayName="OpenTelemetry Collector"
-// +operator-sdk:csv:customresourcedefinitions:resources={{Pod,v1},{Deployment,v1},{Service, v1}}
+// +operator-sdk:csv:customresourcedefinitions:resources={{Pod,v1},{Deployment,apps/v1},{DaemonSets,apps/v1},{StatefulSets,apps/v1},{ConfigMaps,v1},{Service,v1}}
```

Comment:
Services? Service Accounts? Double-check the objects we create.

Reply:
service is there. again I don't want to list everything, see my previous comment

Reply:
It's fine to not list everything, just state the expectation so that your future self knows when to update this list.